twake/backend/node/src/core/platform/services/database/services/connectors/cassandra.ts

```diff
@@ -2,6 +2,7 @@
 import cassandra from "cassandra-driver";
 import { AbstractConnector } from "./abstract-connector";
 
+
```

Comment:
Please avoid to add blank lines for nothing. A common rule is to have 1 blank line